source/learn/intrinsics/C_index.md

```diff
@@ -0,0 +1,19 @@
+# Procedures for binding to C interfaces
```

Comment:
Just `c.md` seems to short. It is also called C-Fortran-Interoperability, so `.../cfi.md` or `.../cfi/index.md` should work